build.xml

```diff
@@ -57,4 +64,16 @@
       <arg line="${example.args}"/>
     </java>
   </target>
+
+  <target name="send_feed" depends="build"
```

Comment:
Do we want a more gsafeed/records-based name here, to make room to add groups later?